ci-operator/step-registry/aws/deprovision/route53/private-hosted-zone/aws-deprovision-route53-private-hosted-zone-commands.sh

```diff
@@ -22,3 +27,14 @@ CHANGE_ID="$(echo "${HOSTED_ZONE}" | jq -r '.ChangeInfo.Id' | awk -F / '{printf
 aws --region "${REGION}" route53 wait resource-record-sets-changed --id "${CHANGE_ID}" &
 wait "$!"
 echo "AWS route53 hosted zone $(cat "${HOSTED_ZONE_ID}") successfully deleted."
+
+
+if [[ ${ENABLE_SHARED_PHZ} == "yes" ]]; then
+  export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred_shared_account"
```

Comment:
In `aws-ipi-shared-vpc-phz` case, the public dns records are still going to be created, which account would those dns be created in?

Reply:
> DNS Records in private and public zone and cluster are created by Account B

Which account is the public zone located? The `.awscred` one?

Reply:
> Which account is the public zone located? The `.awscred` one?

Yes